src/alignment.js

```diff
@@ -121,7 +121,7 @@ var alignWithDOM = function(nodeName, key, statics) {
 /**
  * Clears out any unvisited Nodes, as the corresponding virtual element
  * functions were never called for them.
- * @param {!Element} node
+ * @param {!Node} node
```

Comment:
Are you sure about that? I thought the only place you could pass `DocumentFrament` in was via `patch`. Any `elementOpen` or `elementVoid` calls would all create `Element`'s no?